notebooks/community/pipelines/google_cloud_pipeline_components_bqml_pipeline_anomaly_detection.ipynb

```diff
@@ -0,0 +1,1510 @@
+{
```

Comment:
<p>This looks awkward using urlparse. Why not just use os.path.join?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Because it is not a path but url. So I prefer to use urlib</p>